spec/models/service_template_transformation_plan_task_spec.rb

```diff
@@ -307,25 +309,21 @@
       before do
         allow(Time).to receive(:now).and_return(time_now)
         allow(conversion_host).to receive(:run_conversion).with(task_1.conversion_options).and_return(
-          {
-            "wrapper_log" => "/tmp/wrapper.log",
-            "v2v_log"     => "/tmp/v2v.log",
-            "state_file"  => "/tmp/v2v.state"
-          }
+          "wrapper_log" => "/tmp/wrapper.log",
+          "v2v_log"     => "/tmp/v2v.log",
+          "state_file"  => "/tmp/v2v.state"
```

Comment:
Were these changes related to your change or just `rubocop -a` ?  If the latter would prefer to keep them out of the same PR as other changes, you can make a separate rubocop fixes PR.  It makes it easier to review and see what is going on.

Reply:
`rubocop -a` might have added them because I did not.

Reply:
Yeah thought so, okay can you drop that second commit and just amend your first with fixes for only what was changed

Reply:
@thearifismail I'll let this go for now but in the future please don't mix rubocop fixes with other code changes in the same PR.  It generally isn't advisable to run rubocop -a on an entire file unless your intent is to put in a rubocop fix only PR because it introduces a lot of unnecessary changes that make the original intent of the PR harder to understand when looking at the diff.